apps/etterlatte-utbetaling/src/main/kotlin/utbetaling/avstemming/KonsistensavstemmingService.kt

```diff
@@ -0,0 +1,146 @@
+package no.nav.etterlatte.utbetaling.avstemming
+
+import no.nav.etterlatte.libs.common.tidspunkt.Tidspunkt
+import no.nav.etterlatte.libs.common.tidspunkt.norskTidssone
+import no.nav.etterlatte.libs.common.tidspunkt.toTidspunkt
+import no.nav.etterlatte.utbetaling.grensesnittavstemming.AvstemmingDao
+import no.nav.etterlatte.utbetaling.grensesnittavstemming.UUIDBase64
+import no.nav.etterlatte.utbetaling.iverksetting.utbetaling.Foedselsnummer
+import no.nav.etterlatte.utbetaling.iverksetting.utbetaling.NavIdent
+import no.nav.etterlatte.utbetaling.iverksetting.utbetaling.Saktype
+import no.nav.etterlatte.utbetaling.iverksetting.utbetaling.UtbetalingDao
+import no.nav.etterlatte.utbetaling.iverksetting.utbetaling.Utbetalingslinje
+import no.nav.etterlatte.utbetaling.iverksetting.utbetaling.UtbetalingslinjeId
+import no.nav.etterlatte.utbetaling.iverksetting.utbetaling.Utbetalingslinjetype
+import java.time.Instant
+import java.time.LocalDate
+import java.time.LocalTime
+import java.time.ZoneId
+import java.time.ZonedDateTime
+
+class KonsistensavstemmingService(private val utbetalingDao: UtbetalingDao, private val avstemmingDao: AvstemmingDao) {
+
+    fun lagKonsistensavstemming(dag: LocalDate, saktype: Saktype): Konsistensavstemming {
+        val loependeYtelseFom = dag.atStartOfDay().toTidspunkt(ZoneId.of("Europe/Oslo"))
+        val registrertFoerTom = dag.minusDays(1).atTime(LocalTime.MAX).toTidspunkt(ZoneId.of("Europe/Oslo"))
+
+        val relevaanteUtbetalinger = utbetalingDao.hentUtbetalingerForKonsistensavstemming(
+            loependeFraOgMed = loependeYtelseFom,
+            opprettetFramTilOgMed = registrertFoerTom,
+            saktype = saktype
+        )
+
+        /**
+         * 1. Ta bort de utbetalingene som er oppdrettet etter konsistensavstemmingstidspunktet
+         * 2. Grupper utbetalinger per sak
+         * 3. Samle en liste av utbetalinger til en samlet utbetaling med mange utbetalingslinjer (ligner på struktur vi
+         *      sender i konsistensavstemmingen)
+         * 4. Filtrerer bort alle oppdragslinjer som ikke er "aktive" i OS på konsistensavstemmingsdatoen
+         * 5. Mapper de sakene med oppdragslinjer igjen om til OppdragForKonsistensavstemming, og luker bort de sakene
+         *      uten noen oppdragslinjer.
+         *
+         * Bada bing bada bom
+         *
+         * Inspirasjon og takk rettes mot su-se-bakover's implementasjon av tilsvarende
+         */
+        val loependeUtbetalinger: List<OppdragForKonsistensavstemming> = relevaanteUtbetalinger
```

Comment:
relev**AA**nteUtbetalinger -> relev**A**nteUtbetalinger ?

Reply:
this shit relevånt yo